internal/block/prefetch_block.go

```diff
@@ -135,12 +135,12 @@ func (pmb *prefetchMemoryBlock) AbsStartOff() int64 {
 
 func (pmb *prefetchMemoryBlock) SetAbsStartOff(startOff int64) error {
 	if startOff < 0 {
-		return fmt.Errorf("startOff cannot be negative, got %d", startOff)
+		return fmt.Errorf("SetAbsStartOff: startOff cannot be negative, got %d", startOff)
 	}
 
 	// If absStartOff is already set, then return an error.
 	if pmb.absStartOff >= 0 {
-		return fmt.Errorf("AbsStartOff is already set, it should be set only once.")
+		return fmt.Errorf("SetAbsStartOff: AbsStartOff is already set, it should be set only once.")
```

Comment:
```suggestion
		return fmt.Errorf("SetAbsStartOff: AbsStartOff is already set.")
```
The second statement seems redundant.